src/components/app-root/index.tsx

```diff
@@ -35,9 +35,9 @@ class AppRootBase extends React.PureComponent<AppRootProps, {}> {
         bottomPane = <LoadData/>;
       } else {
         bottomPane = (
-          <SplitPane split="vertical" defaultSize={200}>
+          <SplitPane split="vertical" defaultSize={200} minSize={100} maxSize={200}>
             <DataPane/>
-            <SplitPane split="vertical" defaultSize={235}>
+            <SplitPane split="vertical" defaultSize={235} minSize={117} maxSize={235}>
```

Comment:
The min size is too small, which allows user to construct a broken state

![image](https://user-images.githubusercontent.com/111269/31857676-c21764f4-b70e-11e7-8244-90dfba181fa1.png)